openoa/utils/imputing.py

```diff
@@ -200,12 +200,12 @@ def impute_all_assets_by_correlation(
                 # target_data=data.xs(target_id, level=1).loc[:, [impute_col]],
                 target_data=data.loc[
                     data.index.get_level_values(1) == target_id, [impute_col]
-                ].droplevel("id"),
+                ].droplevel("WTUR_TurNam"),
```

Comment:
One potential issue here is that we are assuming imputing will only be performed on turbine data, whereas previously it could have been applied to met tower data. Would it make sense to make the id column an argument that defaults to "WTUR_TurNam". Or we could just use "id" instead of "WTUR_TurNam" in the SCADA data to avoid issues like this.